grails-app/services/org/pih/warehouse/inventory/RequisitionStatusTransitionEventService.groovy

```diff
@@ -0,0 +1,31 @@
+/**
+* Copyright (c) 2012 Partners In Health.  All rights reserved.
+* The use and distribution terms for this software are covered by the
+* Eclipse Public License 1.0 (http://opensource.org/licenses/eclipse-1.0.php)
+* which can be found in the file epl-v10.html at the root of this distribution.
+* By using this software in any fashion, you are agreeing to be bound by
+* the terms of this license.
+* You must not remove this notice, or any other, from this software.
+**/
+package org.pih.warehouse.inventory
+
+import org.codehaus.groovy.grails.commons.GrailsApplication
+import org.pih.warehouse.core.MailService
+import org.pih.warehouse.requisition.Requisition
+
+
+class RequisitionStatusTransitionEventService {
+
```

Comment:
please add
```groovy
static transactional = true
```
so that we remember to mark this service as Transactional when pulling it into Grails 3 branch

Reply:
Is there a reason that it should be transactional in the current form in Grails 3? 🧐